android-client-sdk/src/main/java/com/devcycle/sdk/android/api/EventQueue.kt

```diff
@@ -189,6 +202,10 @@ internal class EventQueue constructor(
     }
 
     private fun run() {
+        if(disableEventLogging){
+            DVCLogger.i("DVC Event Logging disabled, skipping this call")
```

Comment:
this would be a really annoying log if you've got events disabled. It would be logging at info level every time you evaluate a variable or do anything else that triggers events.

The highest level I'd make this is "debug", probably even "verbose"